devops/charts/redis/templates/statefulset.yaml

```
@@ -53,20 +53,18 @@ spec:
                   - -c
                   - redis-cli --user $REDISCLI_USER -h $HOSTNAME shutdown save
           livenessProbe:
-            exec:
-              command:
-                - /bin/sh
-                - -c
-                - test $(redis-cli --user $REDISCLI_USER -h $HOSTNAME ping) == PONG
-            initialDelaySeconds: 10
+            tcpSocket:
+              port: 6379
+            initialDelaySeconds: 5
+            periodSeconds: 10
             timeoutSeconds: 1
             failureThreshold: 3
           readinessProbe:
             exec:
               command:
                 - /bin/sh
                 - -c
-                - test $(redis-cli --user $REDISCLI_USER -h $HOSTNAME ping) == PONG
+                - '[ "$(redis-cli --user "$REDISCLI_USER" -h "$HOSTNAME" ping)" = "PONG" ] || exit 1'
```

Should be a `==`, no?
```suggestion
                - '[ "$(redis-cli --user "$REDISCLI_USER" -h "$HOSTNAME" ping)" == "PONG" ] || exit 1'
```

---

Ah, just looked it up to check, `=` is actually better apparently. Ignore me